doc/lightning-close.7.txt

```diff
@@ -0,0 +1,58 @@
+LIGHTNING-CLOSE(7)
+==================
+:doctype: manpage
+
+NAME
+----
+lightning-close - Protocol for closing channels with direct peers
+
+SYNOPSIS
+--------
+*close* 'id' ['force'] ['timeout']
+
+DESCRIPTION
+-----------
+
+The *close* RPC command attempts to close the channel cooperatively
+with the peer.
+It applies to the active channel of the direct peer corresponding to
+the given peer 'id'.
+
+The *close* command will time out and return with an error when the
+number of seconds specified in 'timeout' is reached.
+If unspecified, it times out in 30 seconds.
+
+The 'force' argument, if the JSON value 'true', will cause the
+channel to be unilaterally closed when the timeout is reached.
+If so, timeout will not cause an error, but instead cause the
+channel to be failed and put onchain unilaterally.
+Unilateral closes will lead to your funds getting locked according
+to the 'to_self_delay' parameter of the peer.
+
+Normally the peer needs to be live and connected in order to negotiate
+a bilateral close.
+Forcing a unilateral close can be used if you suspect you can no longer
+access the peer.
+
+RETURN VALUE
+------------
+
+On success, an object with fields 'tx' and 'txid' containing the
+closing transaction are returned.
+It will also have a field 'type' which is either the JSON string
+'bilateral' or the JSON string 'unilateral'.
```

Comment:
I usually use collaborative close to describe the bilateral close, which is a bit nicer. The spec talks about a "mutual close" or "cooperative close" which is also slightly nicer than bilateral (and you use cooperative in the code later).

Reply:
Hmm.  Grepping `lightning-rfc`:

* cooperative: 1 time, in 00-introduction.md
* bilateral: 0 times, ha
* mutual: many times, in 00-introduction.md, 02-peer-protocol.md, 05-onchain.md

Perhaps we should go with 'mutual' instead, as that seems to be the preferred term in the RFC.  Note that the BOLT spec uses 'mutual' and 'unilateral'.